QuizApp/App/Screens/Quiz details/QuizDetailsViewController.swift

```diff
@@ -162,7 +162,9 @@ private extension QuizDetailsViewController {
         imageView.kf.setImage(with: imageUrl)
     }
 
-    @objc func leaderboardButtonTapped() { }
+    @objc func leaderboardButtonTapped() {
+        viewModel.goToLeaderboard()
+    }
```

Comment:
Since you're messing with this part of the code, you can refactor this to use combine.